.vale.ini

```diff
@@ -1,9 +1,9 @@
 StylesPath = styles
-MinAlertLevel = suggestion # suggestion, warning or error
+MinAlertLevel = warning # suggestion, warning or error
 # Only Markdown and .txt files; change to whatever you're using.
 [*.{md}]
 # List of styles to load.
-BasedOnStyles = Dxw,Govuk
+BasedOnStyles = Dxw,Govuk,,write-good,proselint
```

Comment:
Is there an invisible style between `Govuk` and `write-good`?

Reply:
you'd think the linter would catch that :D